src/pages/PageUsingComponents.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { SplitClient, SplitTreatments } from '@splitsoftware/splitio-react';
+import { feature_1, feature_2, feature_3 } from '../sdkConfig';
+
+/* This example showcasts SplitClient and SplitTreatments components */
+
+const Loading = () => {
+  return <div>Loading SDK...</div>
+}
+
+export default () => {
+  return (
+    <main>
+
+      <SplitTreatments names={[feature_1]} >
+        {({ treatments }) => {
```

Comment:
Add a comment, being brief about the treatments prop, what it has. Mention that it's provided to render props.